tests/parse_config_update_test.py

```diff
@@ -170,3 +173,53 @@ def test_parse_streams_skips_stream_info_if_remove_config_and_schema_present_but
     config_message = deserialise_rf5k(message)
     streams = tuple(_parse_streams(CommandType.REMOVE, config_message.streams))
     assert not streams
+
+
+@patch(
+    "forwarder.parse_config_update.deserialise_rf5k",
+    side_effect=RuntimeError("Runtime Error"),
+)
+def test_parse_config_update_when_runtime_error_occurs(mock_func):
+    test_channel_name = "test_channel"
+    message = serialise_rf5k(
+        UpdateType.ADD,
+        [StreamInfo(test_channel_name, "f142", "output_topic", Protocol.PVA)],
+    )
+    config_update = parse_config_update(message)
+    assert config_update.command_type == CommandType.INVALID
+
+
+@patch(
+    "forwarder.parse_config_update.deserialise_rf5k",
+    side_effect=flatbuffer_struct.error("Flatbuffer Error"),
+)
+def test_parse_config_update_when_flatbuffer_struct_error_occurs(mock_func):
```

Comment:
These tests need better names as they should act as documentation of the system.
Perhaps something like "command is invalid on flatbuffer struct error"

Reply:
Can you check if the names are ok now?